base/applications/mspaint/drawing.cpp

```diff
@@ -91,14 +91,35 @@ Poly(HDC hdc, POINT * lpPoints, int nCount,  COLORREF fg,  COLORREF bg, int thic
     SetROP2(hdc, oldRop);
 }
 
+double GetDistancePow2(POINT p1, POINT p2)
```

Comment:
Can return `LONG` instead (see comment below). Also, it could be marked `inline`.

Reply:
At first sight seems it is returning LONG * LONG + LONG * LONG, can we be sure that those LONGs are small enough that the result will fit in a LONG too?

Reply:
Fixed.

Reply:
5000 * 5000 == 25000000.
`MAXLONG` == 4294967295.
Good enough!